submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,87 @@
+const url = 'https://randomuser.me/api/?results=12';
+const content = document.querySelector('.articleBlock');
+const form = document.forms;
+let friends = [];
+let friendsCopy = [];
+
+const data = (url) => fetch(url)
+    .then(handleErrors)
+    .then(res => res.json())
+    .then(res => res.results)
```

Comment:
Getting a property from an object is not an asynchronous task, you don't need another `.then` after this one.